csharp/private/rules/library.bzl

```diff
@@ -91,6 +92,9 @@ csharp_library = rule(
             doc = "The standard set of assemblies to reference.",
             default = "@net//:StandardReferences",
         ),
+        "internals_visible_to": attr.string_list(
+            doc = "C# libraries that can see our internal symbols. Using this rather than the InternalsVisibleTo assembly attribute will improve build caching.",
```

Comment:
```suggestion
            doc = "Other C# libraries that can see the assembly's internal symbols. Using this rather than the InternalsVisibleTo assembly attribute will improve build caching.",
```